packages/mobile/src/screens/profile-screen/ProfileInfo.tsx

```diff
@@ -62,26 +62,38 @@ const messages = {
 }
 
 type ProfileInfoProps = {
-  profile: User
   onFollow: () => void
 }
 
 export const ProfileInfo = (props: ProfileInfoProps) => {
-  const { profile, onFollow } = props
+  const { onFollow } = props
   const styles = useStyles()
-  const { does_current_user_follow, does_follow_current_user } = profile
-  const accountUser = useAccountUser()
-  const isOwner = accountUser?.user_id === profile.user_id
+  const profile = useSelectProfile([
+    'user_id',
+    'name',
+    'handle',
+    'does_current_user_follow',
+    'does_follow_current_user'
+  ])
+
+  const {
+    name,
+    handle,
+    does_current_user_follow,
+    does_follow_current_user
+  } = profile
+
+  const isOwner = useSelectorWeb(getIsOwner)
```

Comment:
we could just match on handle equality here instead of a useSelectorWeb right? that would save 1 rerender

Reply:
so how do we get the owner's handle/id, just do `useSelectorWeb(getUserId)`?

Reply:
oh nvm ... i misread. you do need that somehow. it is something we don't need to constantly pull and maybe could be in RN state itself, but nvm!!!!!!